java/java.lsp.server/test/unit/src/org/netbeans/modules/java/lsp/server/protocol/ServerTest.java

```diff
@@ -2212,7 +2209,7 @@ public void logMessage(MessageParams arg0) {
         assertEquals("\n" +
                      "    @Override\n" +
                      "    public void run() {\n" +
-                     "        throw new UnsupportedOperationException(\"Not supported yet.\"); //To change body of generated methods, choose Tools | Templates.\n" +
+                     "        throw new UnsupportedOperationException(\"Not supported yet.\"); // Generated from nbfs://nbhost/SystemFileSystem/Templates/Classes/Code/GeneratedMethodBody\n" +
```

Comment:
How comes this hasn't shown up before?

Reply:
Good question. I would expect the `ServerTest` to be falling since https://github.com/apache/netbeans/commit/3db24dc9aea3c76aed71efb129e9af2e7b345c8e.